addOns/exim/src/main/java/org/zaproxy/addon/exim/urls/UrlsImporter.java

```diff
@@ -0,0 +1,141 @@
+/*
+ * Zed Attack Proxy (ZAP) and its related class files.
+ *
+ * ZAP is an HTTP/HTTPS proxy for assessing web application security.
+ *
+ * Copyright 2021 The ZAP Development Team
```

Comment:
This is a new class though contains the majority of the relevant original code, should I maintain 2014?

Reply:
Well, the code needs to be tidied up. I left some comments if you don't mind doing that in this PR.